packages/metro-file-map/src/watchers/__tests__/integration-test.js

```diff
@@ -207,6 +207,33 @@ describe.each(Object.keys(WATCHERS))(
       });
     });
 
+    maybeTest('detects changes to files in a new directory', async () => {
```

Comment:
I assume you're planning to add this test on `main`? Or do we have equivalent coverage there by now?

Reply:
Yep - diff incoming.